modules/gitops-creating-a-new-client-using-keycloak.adoc

```diff
@@ -73,6 +73,57 @@ spec:
       rootCA: ""
 ----
 
+* Optional: Customize the route name of the Keycloak instance by adding a host name attribute in the Argo CD CR. Use this feature to support advanced routing use cases such as balancing incoming traffic load among multiple link:https://docs.openshift.com/container-platform/latest/networking/ingress-sharding.html#nw-ingress-sharding_ingress-sharding[Ingress Controller shards].
++
+** To add a `host name` attribute in the Argo CD CR, add this YAML snippet in your namespace.
++
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  labels:
+    example: route
+spec:
+  sso:
+    provider: keycloak
+    keycloak:
+     host: sso.test.example.com #<1>
+  server:
+    ingress:
+      enabled: true
+    insecure: true
+----
+<1> Specifies the name of the host key
++
+** To apply the YAML, run the following command in the {gitops-shortname} CLI:
++
+[source,terminal]
+----
+kubectl apply -f argocd.yaml -n <any-namespace>
+----
++
+** You can view that the following `host name` attribute is added in the `keycloak` route. 
++
```

Comment:
It is mentioned here that the hostname attribute is added in Keycloak, but it does not explain how to verify the output. You can add this command: `kubectl get route keycloak -n <any-namespace> -o yaml` to allow the user to check the output and see if it has been updated.

Reply:
here you can add this `oc get route keycloak -n <any-namespace> -o yaml`

Reply:
Fixed.